meta-iot2000-example/recipes-example/iot2000setup/files/iot2000setup.py

```diff
@@ -18,6 +18,10 @@
 wifiInterfaceConfigured = True
 deviceIsIot2020 = False
 
+class ansicolors:
+	clear = '\033[2J'
+	blueBegin = '\033[1;34m'
+	blueEnd = '\033[0m'
```

Comment:
Just nitpicking but this is not only applicable to end blue coloring :)
`reset` or `normal` would be a better name IMHO.